microprofile.jdt/pom.xml

```diff
@@ -3,14 +3,14 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.eclipse.lsp4mp</groupId>
   <artifactId>parent</artifactId>
-  <version>0.6.0-SNAPSHOT</version>
+  <version>0.7.0-SNAPSHOT</version>
   <packaging>pom</packaging>
   <name>microprofile.jdt.ls :: parent</name>
   <description>microprofile.jdt.ls parent</description>
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <project.version>0.6.0-SNAPSHOT</project.version>
+    <project.version>0.7.0-SNAPSHOT</project.version>
```

Comment:
Just wondering if this is needed. Should the references not use 0.7.0-SNAPSHOT anyways because that's set at the top level ?

Reply:
Doesn't look like it, I'll remove